.gitmodules

```diff
@@ -9,3 +9,6 @@
 	path = core/packages/contracts/lib/openzeppelin-contracts
 	url = https://github.com/openzeppelin/openzeppelin-contracts
 	branch = v4.8.2
+[submodule "cumulus"]
+	path = cumulus
+	url = https://github.com/Snowfork/cumulus
```

Comment:
Don't we need to specify the branch too?

Reply:
I think we do not need,  PR here already contains matched [commit](https://github.com/Snowfork/cumulus/tree/07296e8f0f3bcaabd11aeb6e99a38e08ecbb1da2) in cumulus.